From 76cffdd94b8f01afe38c5323737e3c93da041fa6 Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo+renesas@jmondi.org>
Date: Thu, 22 Feb 2018 04:47:19 -0500
Subject: [PATCH 119/437] media: MAINTAINERS: Add entry for Techwell TW9910

Add entry for Techwell TW9910 video decoder. The driver is currently
orphaned.

Signed-off-by: Jacopo Mondi <jacopo+renesas@jmondi.org>
Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
---
 MAINTAINERS | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 3cbebe7635ce..4c912bbdb405 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -13673,6 +13673,12 @@ L:	linux-media@vger.kernel.org
 S:	Maintained
 F:	drivers/media/rc/ttusbir.c
 
+TECHWELL TW9910 VIDEO DECODER
+L:	linux-media@vger.kernel.org
+S:	Orphan
+F:	drivers/media/i2c/tw9910.c
+F:	include/media/i2c/tw9910.h
+
 TEE SUBSYSTEM
 M:	Jens Wiklander <jens.wiklander@linaro.org>
 S:	Maintained
-- 
2.16.3

